packages/roosterjs-editor-plugins/lib/PasteManager/PasteManager.ts

```diff
@@ -9,6 +9,7 @@ import {
 import { Editor, EditorPlugin } from 'roosterjs-editor-core';
 import { processImages } from './PasteUtility';
 import { fromHtml, unwrap } from 'roosterjs-editor-dom';
+import { browserData } from 'roosterjs-editor-core';
```

Comment:
You could merge the import with the first one since they're all imported from 'rootster-editor-core'